Sources/WinAppDriver/Win32ProcessTree.swift

```diff
@@ -64,23 +79,53 @@ internal class Win32ProcessTree {
         return jobHandle
     }
 
-    private static func createProcessInJob(commandLine: String, jobHandle: HANDLE) throws -> HANDLE {
+    private static func createProcessInJob(
+        commandLine: String,
+        jobHandle: HANDLE,
+        options: ProcessLaunchOptions = ProcessLaunchOptions()
+    ) throws -> HANDLE {
         try commandLine.withCString(encodedAs: UTF16.self) { commandLine throws in
             var startupInfo = STARTUPINFOW()
             startupInfo.cb = DWORD(MemoryLayout<STARTUPINFOW>.size)
+            var redirectStdHandle = false
+
+            let creationFlags =
+                DWORD(CREATE_SUSPENDED) | DWORD(CREATE_NEW_PROCESS_GROUP)
+                | (options.spawnNewConsole ? DWORD(CREATE_NEW_CONSOLE) : 0)
+            if let stdoutHandle = options.stdoutHandle {
+                startupInfo.hStdOutput = stdoutHandle
+                redirectStdHandle = true
+            } else {
+                startupInfo.hStdOutput = INVALID_HANDLE_VALUE
```

Comment:
We didn't need to set this before.  Why now?  Are you sure this is the right thing to do?  Before, I presume, it was 0, but now it's F-bazillion, which might cause problems, because that's the pseudo-handle `GetCurrentProcess()` returns.  Also, the docs say:

> The caller is responsible for ensuring that the standard handle fields in [STARTUPINFO](https://learn.microsoft.com/en-us/windows/desktop/api/processthreadsapi/ns-processthreadsapi-startupinfow) contain valid handle values. These fields are copied unchanged to the child process without validation, even when the dwFlags member specifies STARTF_USESTDHANDLES. Incorrect values can cause the child process to misbehave or crash.

Note it says "valid handles".  I'm really not sure what the right thing to do here is.